### DISCUSSION TITLE
How To Set Subdomain Cookie

### DISCUSSION THREAD

**Discussion by ramonmalcolm10** [sveltejs/kit#10164] [Q&A] [answered]
How to set cookie from main domain that is accessible from subdomain?

**Comment by ghostdevv** [comment, marked as answer]
I believe `domain` field

SEE:
https://stackoverflow.com/questions/18492576/share-cookies-between-subdomain-and-domain#23086139
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie
https://www.npmjs.com/package/cookie

**Reply by ramonmalcolm10** [reply]
Thanks
